content/en/docs/collector/deployment/single-writer-principle.md

```diff
@@ -0,0 +1,57 @@
+---
+title: The Single Writer Principle
+description: How to handle the single writer principle in Collector deployments
+weight: 1
+---
+
```

Comment:
As the doc is likely not self-contained, consider adding links from other topics.